youtube_dl/extractor/porn91.py

```diff
@@ -36,28 +39,28 @@ def _real_extract(self, url):
             raise ExtractorError('91 Porn says: Daily limit 10 videos exceeded', expected=True)
 
         title = self._search_regex(
-            r'<div id="viewvideo-title">([^<]+)</div>', webpage, 'title')
+            r'<h4 class="login_register_header"[^>]+>([^<]+)</h4>', webpage, 'title')
         title = title.replace('\n', '')
 
         video_link_url = self._search_regex(
-            r'<textarea[^>]+id=["\']fm-video_link[^>]+>([^<]+)</textarea>',
+            r'document\.write\(strencode2\("([^"]+)"\)\);',
```

Comment:
Similarly, valid JS could have whitespace around `[.(),;]`, or use '' instead of "".